packages/narwallets/src/lib/injected-narwallets.ts

```diff
@@ -0,0 +1,134 @@
+// Empty string if we haven't signed in before.
```

Comment:
Is this file relevant, are you making use of these types are these the correct types that your wallet uses they seem to be a copy of the Sender wallet types?